scripts/update/Updater.php

```diff
@@ -247,6 +247,6 @@ public function update($initialVersion)
             $this->setVersion('1.2.0');
         }
 
-        $this->skip('1.2.0', '2.1.2');
+        $this->skip('1.2.0', '2.1.3');
```

Comment:
Same note about versions and dependencies

Reply:
https://github.com/oat-sa/extension-tao-publishing/pull/61#discussion_r393203588